package.json

```diff
@@ -148,7 +148,8 @@
       "react/self-closing-comp": "error",
       "array-callback-return": "error",
       "react-hooks/rules-of-hooks": "error",
-      "react-hooks/exhaustive-deps": "error"
+      "react-hooks/exhaustive-deps": "error",
+      "prettier/prettier": [1, { "endOfLine": "auto"}]
```

Comment:
Btw, do we still need this if we have the `. prettierrc` file?

Reply:
i tried specifying in the .prettierrc file, but it does not work

Reply:
Haha I just found out the reason in my very own comment (since I copied and pasted the filename). See how there is an extra space?

```text
. prettierrc
```

Reply:
now that i did it, there are multiple linting file changes. i guess that since all this way, the prettierrc file has been inactive, so was the linting.

Reply:
can i do the fix for the prettierrc in another pr?

Reply:
Yeah I think the linting can be another PR after this is merged